packages/flutter/lib/src/cupertino/text_theme.dart

```diff
@@ -248,11 +248,27 @@ class CupertinoTextThemeData extends Diagnosticable {
       dateTimePickerTextStyle ?? _dateTimePickerTextStyle,
     );
   }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    const CupertinoTextThemeData defaultData = CupertinoTextThemeData();
```

Comment:
Getting `error • Evaluation of this constant expression throws an exception • packages/flutter/lib/src/cupertino/text_theme.dart:255:48 • const_eval_throws_exception` but the analyzer won't let me change to `final` either.

Reply:
Is it because you're missing required parameters (this assertion https://github.com/flutter/flutter/pull/44317/files#diff-6cb3174a51ae9b800fc6377dfc24234cR161)?

Reply:
`primaryColor` has a default value `CupertinoColors.systemBlue`. The weird thing is I'm able to compile apps and they run without throwing, despite this analyzer error.